### PR TITLE
Implement `has_one_value()`

### PR DESCRIPTION
**Pull request by etiennebacher**
Coming from https://github.com/easystats/datawizard/pull/318

### CODE REVIEW

**Comment by strengejacke** [comment]
Just one thing... Should we have an `na.rm` argument, just in case someone wants to test only for non-missing values?

**Comment by etiennebacher** [comment]
I'm not sure, wouldn't it be better to explicitly remove the NA from the vector you want to use in `has_one_value()`?

File: NAMESPACE [hunk 1]
```diff
@@ -1564,6 +1564,7 @@ export(get_variance_residual)
 export(get_variance_slope)
 export(get_weights)
 export(has_intercept)
+export(has_one_value)
```
Comment (IndrajeetPatil) [inline NAMESPACE:1567]: Maybe `has_single_value()` is a better name? WDYT?

Comment (etiennebacher) [reply NAMESPACE:1567]: Agreed (but can't modify now, I will change it later)

**Pull request merged**